site2/website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "2.6.0",
```

Comment:
should keep 2.5.2 as default view, and change to 2.6.0 after the index is build.